src/linker/Linker.Dataflow/DynamicallyAccessedMembersBinder.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Mono.Cecil;
+
+namespace Mono.Linker.Dataflow
```

Comment:
Nit: Maybe we should put this into `Mono.Linker` namespace as it's not DataFlow-only anymore... but I'm fine either way

Reply:
Yep, this should go to something like `Mono.Linker.TypeDefinitionLookup` class and possibly changed to extension methods

Reply:
Changed to `Mono.Linker` with extension methods. I kept the class name because this is specific to the `DynamicallyAccessedMemberTypes`, and doesn't have other member lookup logic.